projects/frontend/shared-components/gui/sonar-project.properties

```diff
@@ -0,0 +1,14 @@
+sonar.projectKey=taurus.shared-components
```

Comment:
Most likely we are going to continue using codacy in favour of introducing sonar in parallel. We can drop this file for now, and revise in a dedicated effort for code quality check.